pkg/webhook/validator.go

```diff
@@ -490,8 +490,53 @@ func ociSignatureToPolicySignature(ctx context.Context, sigs []oci.Signature) []
 				},
 			})
 		} else {
-			// TODO(mattmoor): Is there anything we should encode for key-based?
-			ret = append(ret, PolicySignature{})
+			ret = append(ret, PolicySignature{
+				// TODO(mattmoor): Is there anything we should encode for key-based?
+			})
+		}
+	}
+	return ret
+}
+
+type attestation struct {
+	oci.Signature
+
+	PredicateType string
+	Payload       []byte
+}
+
+func attestationToPolicyAttestation(ctx context.Context, atts []attestation) []PolicyAttestation {
```

Comment:
nit: since it takes array, I'd think attestationToPolicyAttestations would be more appropriate.

Reply:
done